helm/fmtok8s-email-service/templates/prometheussvc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  {{- if .Values.service.name }}
```

Comment:
@salaboy I think that this condition is not necessary.
In which scenario is necessary it?

Reply:
@mcruzdev yeah.. i think it is good to add the condition